README.md

```diff
@@ -1,14 +1,27 @@
 # SPA website
 
-This is the jekyll project for programme chairs to produce most of the pages associated with the [http://spaconference.org](SPA conference) website.
+Produce the static pages for the website of [SPA: Software in Practice](http://spaconference.org).
 
-There are some pages that cannot be edited from here:
+The non-static parts of the site are:
 
  - The programme. This is worked on by the organisers privately and published via a different route when ready.
- - Previous conferences, which are left in situ once the year changes. There is a backup of them in the [SPA conference organisation](https://github.com/spaconference/previous-spa-sites).
  - Scripts under MySpa. The PHP functionality of the site is in another (currently private) repo.
 
-To run locally: `bundle exec jekyll serve`. The output of that command will tell you where the site is running locally (something like `localhost:4000/$conference_year/`).
+In addition, previous conference sites are left in situ once the year changes. There is a back up of them in the [SPA conference organisation](https://github.com/spaconference/previous-spa-sites) or they can be seen from the [current SPA site](http://spaconference.org).
+
+# To run locally
+
+You will need [jekyll](https://jekyllrb.com/docs/quickstart/) to run the site locally, and [html-proofer](https://rubygems.org/gems/html-proofer/) to run the tests locally.
+
+Follow the commands in [deploy.sh](/deploy.sh).
```

Comment:
Did you mean `/scripts/deploy.sh' here? That's just an rsync command though, which doesn't seem helpful for running it locally.

Reply:
You are correct! I did not. I meant `scripts/build.sh`. Good spot. I'll update it after lunch (unless you do)

Reply:
Done.

Reply:
Nope, just realised that doesn't make sense either. I'll raise a PR with the actual instructions. Sorry!

Reply:
https://github.com/spaconference/spa-website/pull/17. If you're happy with the instructions there, please go ahead and merge! Thanks